piksi_tools/console/settings.yaml

```diff
@@ -404,6 +404,21 @@
     for each timestamp. When "No DGNSS" is chosen, no differential output will be
     attempted by the receiver.
 
+- group: solution
+  name: process_noise_motion_mode
```

Comment:
Have you discussed with Alex on this? `dynamic motion model'` sounds indeed more customer friendly